### DISCUSSION TITLE
Standard Export with Better BibLatex

### DISCUSSION THREAD

**Discussion by MightyMirko** [retorquere/zotero-better-bibtex#2775] [Q&A]
Hi Dudes, 

somehow my Standard Entries in zotero are not exporting the key "organization". 

Settings:
![grafik](https://github.com/retorquere/zotero-better-bibtex/assets/23053791/c697ce48-6cc6-4c67-9d08-4ad443c92376)

Item in Zotero:
![grafik](https://github.com/retorquere/zotero-better-bibtex/assets/23053791/fba4b398-f3fc-4161-b4d6-64c02fb03051)

Output (references.bib)

```
@standard{din15066RoboterUnd2017,
  type = {Spec (keine Norm)},
  title = {Roboter Und {{Robotikgeräte}} - {{Kollaborierende Roboter}}},
  shorttitle = {{{DIN ISO}}/{{TS}} 15066},
  date = {2017-04},
  number = {15066},
  publisher = {{Beuth Verlag}},
  url = {https://www.beuth.de/de/vornorm/iso-ts-15066/250504228},
  urldate = {2023-11-13},
  pubstate = {gültig},
  version = {2017-04},
  keywords = {\_st/infos\_vollständig},
  file = {C\:\\Users\\mirko\\Zotero\\storage\\EE5TX5FQ\\2017_din_15066_-_roboter_und_robotikgeräte_-_kollaborierende_roboter.pdf;C\:\\Users\\mirko\\Zotero\\storage\\F98JP8ND\\250504228.html}
}
```

Other entries than standard are exporting it. 

Perhaps related: https://github.com/retorquere/zotero-better-bibtex/issues/2543

**Comment by retorquere** [comment]
Please send a debug log

**Reply by retorquere** [reply]
Do you have an MWE of a document with a `@standard` entry with an `organization` field that renders? I haven't been able to construct one and the biblatex manual doesn't list what fields it should/can have. Of `organization` it only says that `@manual`, `@online` or conference-related entry types will have one. I've looked in the `biber` config, but that only lists the default fields for `@standard`, and `organization` isn't in the default fields. According to the biber config, these types get an `organization` field:

* `@dataset`
* `@manual`
* `@misc`
* `@software`
* `@online`
* `@mvproceedings`
* `@proceedings`
* `@inproceedings`

**Reply by retorquere** [reply]
@MightyMirko I will need more information, otherwise I will have to close this issue.

**Comment by MightyMirko** [comment]
I'll need some time for that.
